### DISCUSSION TITLE
Has bromite's development stopped?

### DISCUSSION THREAD

**Discussion by pitsi** [bromite/bromite#1370] [General]
Hello everyone. This is my first time starting a _discussion_ here in github, so please excuse any mistakes.

I am here to ask what is mentioned in the title. Last commit on the repo is from August 5th, the date v92.0.4515.134 was released. 
Since then, there have been 2 more minor updates for chrome's and chromium's v92 (.159 on Aug 16th and .166 on Aug 30th) and the major update to v93 on August 31st. 
https://chromereleases.googleblog.com/search/label/Chrome%20for%20Android

Usually, bromite would receive the update 1-2 days after that, but so far it has been 10+ days since the release of v93 and still nothing, not even a commit.
If the development has indeed stopped, for me it will be the 2nd time in 3 years that I will be looking for a browser.

**Comment by uazo** [comment]
> If the development has indeed stopped

absolutely no

@csagan5 do you need help with the 93.0.4577.62 ? I'm currently working on it, I am missing some crash bugs to fix, but it compile
let me know if you want to open a branch

**Reply by csagan5** [reply]
No help needed, I have been testing [your user scripts PR](https://github.com/bromite/bromite/pull/857).

**Reply by uazo** [reply]
do you have "Add option to force tablet UI" crash at startup?

with flag enabled [my v93 bromite](https://github.com/uazo/bromite/tree/v93/build/patches) crash with
```
--------- beginning of crash
09-11 08:19:17.333  9503  9503 E AndroidRuntime: FATAL EXCEPTION: main
09-11 08:19:17.333  9503  9503 E AndroidRuntime: Process: org.chromium.chrome.stable, PID: 9503
09-11 08:19:17.333  9503  9503 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.chromium.chrome.stable/org.chromium.c                                                                                hrome.browser.ChromeTabbedActivity}: android.content.res.Resources$NotFoundException: Resource ID #0x7f08038f
09-11 08:19:17.333  9503  9503 E AndroidRuntime:        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2913)
09-11 08:19:17.333  9503  9503 E AndroidRuntime:        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3048)
```
and floc ui is not disabled

**Reply by csagan5** [reply]
I haven't tried yet.

**Reply by uazo** [reply]
don't worry, it was my mistake, now fixed.
[here](https://github.com/uazo/bromite/tree/v93/build/patches), if you need it, you can find the patches
and in the meantime I'm testing the build servers: [here](https://github.com/uazo/bromite-buildtools/runs/3600554255) passed the build, [here](https://github.com/uazo/bromite-buildtools/runs/3606987611) some of tests run

**Comment by csagan5** [comment]
> Usually, bromite would receive the update 1-2 days after that, but so far it has been 10+ days since the release of v93 and still nothing, not even a commit.

When there are no major bugfixes in Chromium and no bugfixes in Bromite no release is necessary.

**Comment by csagan5** [comment]
> and the major update to v93 on August 31st.
> https://chromereleases.googleblog.com/search/label/Chrome%20for%20Android

Yes, I missed this one, I was busy working on some Bromite feature.

Please note: we are not obliged to deliver anything second any deadline, this is an open source project made in the free time of the contributors.

**Reply by pitsi** [reply]
> Please note: we are not obliged to deliver anything second any deadline, this is an open source project made in the free time of the contributors.

I know that and I respect it. But sometimes it is better to ask/know early than to wait for that... announcement of bad news. 

I was using kiwi before I even knew about bromite and I abandoned it after it stayed without a single update for 4+ months. 
I also abandoned debian's chromium last year, when it got no updates for ~2 months. It was not until ~6 months later, and no updates during that time, that someone admitted its bad maintenance status and someone else stepped up to help. At that time, debian's chromium had ~140 open cves!
Sadly, this is repeated this year too (with "only" 100 open cves right now) and, no offence, I am glad I left it. Why?
Because nowadays a web browser is a core app everyone uses and every single security hole that is left unpatched, due to lack of maintenance, is leaving the user exposed to risks. I would not mind if it was a media player for instance.

**Reply by csagan5** [reply]
> Because nowadays a web browser is a core app everyone uses and every single security hole that is left unpatched, due to lack of maintenance, is leaving the user exposed to risks.

That is a valid concern; still, only thing you can do is switch to a company-backed browser (for example Brave or Google Chrome) or help the open source forks like this one.

**Reply by pitsi** [reply]
I do my best to help opensource projects and the people behind them, but since I don't write code in any language, all I can do is testing and suggestions for corrections (on occasions that I understand the code a bit). Sorry :(
